[2025-06-11T12:32:25.969+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: multi_region_transform_staging_v2.process_region_asia.process_marketing_campaigns_asia manual__2025-06-11T12:30:24.798327+00:00 [queued]>
[2025-06-11T12:32:25.996+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: multi_region_transform_staging_v2.process_region_asia.process_marketing_campaigns_asia manual__2025-06-11T12:30:24.798327+00:00 [queued]>
[2025-06-11T12:32:25.997+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-06-11T12:32:26.028+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): process_region_asia.process_marketing_campaigns_asia> on 2025-06-11 12:30:24.798327+00:00
[2025-06-11T12:32:26.044+0000] {standard_task_runner.py:60} INFO - Started process 53748 to run task
[2025-06-11T12:32:26.048+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2025-06-11T12:32:26.056+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'multi_region_transform_staging_v2', 'process_region_asia.process_marketing_campaigns_asia', 'manual__2025-06-11T12:30:24.798327+00:00', '--job-id', '1756', '--raw', '--subdir', 'DAGS_FOLDER/master_dag.py', '--cfg-path', '/tmp/tmpu5syudhe']
[2025-06-11T12:32:26.059+0000] {standard_task_runner.py:88} INFO - Job 1756: Subtask process_region_asia.process_marketing_campaigns_asia
[2025-06-11T12:32:26.190+0000] {task_command.py:423} INFO - Running <TaskInstance: multi_region_transform_staging_v2.process_region_asia.process_marketing_campaigns_asia manual__2025-06-11T12:30:24.798327+00:00 [running]> on host 8b0a421f0af7
[2025-06-11T12:32:26.451+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='multi_region_transform_staging_v2' AIRFLOW_CTX_TASK_ID='process_region_asia.process_marketing_campaigns_asia' AIRFLOW_CTX_EXECUTION_DATE='2025-06-11T12:30:24.798327+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-11T12:30:24.798327+00:00'
[2025-06-11T12:32:26.461+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/master_dag.py", line 188, in process_table
    spark = get_spark_session()
TypeError: get_spark_session() missing 1 required positional argument: 'app_name'
[2025-06-11T12:32:26.498+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=multi_region_transform_staging_v2, task_id=process_region_asia.process_marketing_campaigns_asia, execution_date=20250611T123024, start_date=20250611T123225, end_date=20250611T123226
[2025-06-11T12:32:26.611+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 1756 for task process_region_asia.process_marketing_campaigns_asia (get_spark_session() missing 1 required positional argument: 'app_name'; 53748)
[2025-06-11T12:32:26.665+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-11T12:32:26.789+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
